### PR TITLE
[dev-v2.7] Update node-feature-discovery to v0.13.2

### PR DESCRIPTION
**Pull request by thomasferrandiz**
## Issue: 
https://github.com/rancher/rke2/issues/4153

## QA Testing Considerations
node-feature-discovery is not installed directly but only as a dependency of the sriov chart.
The installation procedure is described here: 
https://confluence.suse.com/display/RKE2K3S/CNI+Test+Plan?preview=/1120305984/1120305985/how_to_run_sriov(1).txt

For the purpose of testing only nfd, it is enough to install on one machine and check that the node gets labeled by node-feature-discovery.

The labels added by node-feature-discovery all follow this format:
```
feature.node.kubernetes.io/<feature> = <value>
```

### CODE REVIEW

**Review by rbrtbnfgl — DISMISSED** [review]

**Review by manuelbuil — APPROVED** [review]

**Review by rohitsakala — APPROVED** [review]
LGTM, I wonder if you want to support kube 1.26. 

FYI - The new release of Rancher supports kube 1.26.

File: packages/rancher-nfd/package.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-url: https://github.com/kubernetes-sigs/node-feature-discovery/releases/download/v0.12.0/node-feature-discovery-chart-0.12.0.tgz
+url: https://github.com/kubernetes-sigs/node-feature-discovery/releases/download/v0.13.1/node-feature-discovery-chart-0.13.1.tgz
```
Comment (diogoasouza) [inline packages/rancher-nfd/package.yaml]: Shouldn't this be 0.13.2?

**Review by rohitsakala — CHANGES_REQUESTED** [review]
Please resolve @diogoasouza's comment.

File: packages/rancher-nfd/package.yaml (hunk 1 above)
Comment (thomasferrandiz) [reply packages/rancher-nfd/package.yaml]: yes, PR updated.

**Comment by thomasferrandiz** [comment]
> Please resolve @diogoasouza's comment.

done

**Review by diogoasouza — DISMISSED** [review]

**Review by rohitsakala — DISMISSED** [review]

**Review by rohitsakala — APPROVED** [review]

**Review by diogoasouza — APPROVED** [review]

**Pull request merged**